corehq/apps/app_manager/fixtures/mobile_ucr.py

```diff
@@ -114,7 +114,8 @@ def _get_apps(self, restore_state, restore_user):
         elif (
             toggles.RESTORE_ACCESSIBLE_REPORTS_ONLY.enabled(restore_user.domain)
             and restore_state.params.is_webapps
-            and restore_user.can_view_apps()  # only way to be sure this a web apps restore, not live preview
+            # only way to reliably know that this is a web apps restore, not live preview
+            and not restore_user.can_view_apps(restore_user.domain)
```

Comment:
oof yeah that one's a gotcha.  I wonder what it'd take to make `domain` a required arg here